ai-service/src/ai_service/embeddings/tests/test_encoding.py

```
@@ -0,0 +1,130 @@
+            encoding.embed_query(invalid_input)
+            encoding.embed_documents([invalid_input])
+
+    def test_embed_documents_rejects_empty_list(self):
+        with pytest.raises(errors.EmbeddingError):
+            encoding.embed_documents([])
+
+
+class TestEmbeddingGeneration:
+    """Test embedding generation functionality."""
+
+    def test_embed_query_returns_valid_embedding(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_embed_query_returns_valid_embedding`

Add return type annotation: `None`

(ANN201)